app/controllers/users_controller.rb

```diff
@@ -0,0 +1,31 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    # @user = User.new(username: params[:username], email: params[:email], password: params[:password])
```

Comment:
- [ ] Please remove this commented code. I trust you will do this before merging to the development branch.